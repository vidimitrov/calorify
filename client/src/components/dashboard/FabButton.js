import styled from 'styled-components';
import Button from '@material-ui/core/Button';

const FabButton = styled(Button)`
  && {
    position: fixed;
    bottom: 24px;
    right: 24px;
  }
`;

export default FabButton;
