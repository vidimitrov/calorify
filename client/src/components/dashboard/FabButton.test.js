import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FabButton from './FabButton';

it('Dashboard/FabButton component renders correctly', () => {
  const tree = renderer.create(<FabButton>Click me</FabButton>).toJSON();
  expect(tree).toMatchSnapshot();
});
