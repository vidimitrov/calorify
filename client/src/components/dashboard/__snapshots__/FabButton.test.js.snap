// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard/FabButton component renders correctly 1`] = `
.c0.c0 {
  position: fixed;
  bottom: 24px;
  right: 24px;
}

<button
  className="MuiButtonBase-root-27 MuiButton-root-1 MuiButton-text-3 MuiButton-flat-6 c0"
  disabled={false}
  onBlur={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex="0"
  type="button"
>
  <span
    className="MuiButton-label-2"
  >
    Click me
  </span>
  <span
    className="MuiTouchRipple-root-30"
  />
</button>
`;
